From ffc8fee72abb58a55995320ff92bd7381c246c58 Mon Sep 17 00:00:00 2001
From: Kyle Gospodnetich <me@kylegospodneti.ch>
Date: Wed, 27 Aug 2025 09:30:04 -0700
Subject: [PATCH 1/2] Update to glycin-2

---
 src/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/meson.build b/src/meson.build
index 69902db..404df6a 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -13,8 +13,8 @@ xmlb_dep             = dependency('xmlb', version: '>= 0.3.4')
 yaml_dep             = dependency('yaml-0.1', version: '>= 0.2.5')
 libsoup_dep          = dependency('libsoup-3.0', version: '>= 3.6.0')
 json_glib_dep        = dependency('json-glib-1.0', version: '>= 1.10.0')
-glycin_dep           = dependency('glycin-1', version: '>= 1.0')
-glycin_gtk4_dep      = dependency('glycin-gtk4-1', version: '>= 1.0')
+glycin_dep           = dependency('glycin-2', version: '>= 1.0')
+glycin_gtk4_dep      = dependency('glycin-gtk4-2', version: '>= 1.0')
 
 
 dl_worker_sources = [

From 706cc720cb00ec809c278decb9d467d5b4809ae8 Mon Sep 17 00:00:00 2001
From: Kyle Gospodnetich <me@kylegospodneti.ch>
Date: Wed, 27 Aug 2025 09:41:23 -0700
Subject: [PATCH 2/2] Update glycin include

---
 src/bz-async-texture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bz-async-texture.c b/src/bz-async-texture.c
index 1af6ea3..3cb7942 100644
--- a/src/bz-async-texture.c
+++ b/src/bz-async-texture.c
@@ -23,7 +23,7 @@
 #define MAX_LOAD_RETRIES       3
 #define RETRY_INTERVAL_SECONDS 5
 
-#include <glycin-gtk4-1/glycin-gtk4.h>
+#include <glycin-gtk4-2/glycin-gtk4.h>
 #include <libdex.h>
 
 #include "bz-async-texture.h"
